src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/pipeline/KendraRankingResponseProcessor.java

```diff
@@ -168,7 +173,7 @@ public KendraRankingResponseProcessor create(
             } else {
                 docLimit = Integer.parseInt(inputDocLimit);
             }
-            return new KendraRankingResponseProcessor(tag, description, titleField, bodyField, docLimit, kendraClient);
+            return new KendraRankingResponseProcessor(tag, description, IgnoreFailure, titleField, bodyField, docLimit, kendraClient);
```

Comment:
Same as line 162, should be `ignoreFailure`

Reply:
Thank you!!! changed to lowercase